workflow/rocoto_funcs/setup_xml.py

```diff
@@ -94,6 +97,8 @@ def setup_xml(HOMErrfs, expdir):
         if do_ensemble == "TRUE":
             if os.getenv("DO_IODA", "FALSE").upper() == "TRUE":
                 ioda_bufr(xmlFile, expdir)
+            if os.getenv("DO_ENVAR_RADAR_REF", "FALSE").upper() == "TRUE":
```

Comment:
A quick question: will we use `DO_ENVAR_RADAR_REF` for switching on/off other tasks, such as nonvar cloud analysis task? Thanks!

Reply:
Yes, that is the option to turn on/off the radar reflectivity DA.